src/Scene/Scene.js

```diff
@@ -7,76 +7,139 @@
 /* global window, requestAnimationFrame */
 
 import c3DEngine from 'Renderer/c3DEngine';
-import Globe from 'Globe/Globe';
 import Scheduler from 'Core/Commander/Scheduler';
-import BrowseTree from 'Scene/BrowseTree';
-import NodeProcess from 'Scene/NodeProcess';
-import Quadtree from 'Scene/Quadtree';
-import CoordStars from 'Core/Geographic/CoordStars';
-import Layer from 'Scene/Layer';
-import MobileMappingLayer from 'MobileMapping/MobileMappingLayer';
 import CustomEvent from 'custom-event';
 import StyleManager from 'Scene/Description/StyleManager';
+import LayersConfiguration from 'Scene/LayersConfiguration';
+import Camera from 'Renderer/Camera';
+import * as THREE from 'three';
+import WMTS_Provider from 'Core/Commander/Providers/WMTS_Provider';
+import WMS_Provider from 'Core/Commander/Providers/WMS_Provider';
+import TileProvider from 'Core/Commander/Providers/TileProvider';
+import { STRATEGY_MIN_NETWORK_TRAFFIC } from 'Scene/LayerUpdateStrategy';
 
 var instanceScene = null;
 
 
 const RENDERING_PAUSED = 0;
 const RENDERING_ACTIVE = 1;
 
-function Scene(coordinate, ellipsoid, viewerDiv, debugMode, gLDebug) {
+function Scene(viewerDiv, debugMode, gLDebug) {
     if (instanceScene !== null) {
         throw new Error('Cannot instantiate more than one Scene');
     }
 
-    this.ellipsoid = ellipsoid;
+    this.camera = new Camera(
+        viewerDiv.clientWidth * (debugMode ? 0.5 : 1.0),
+        viewerDiv.clientHeight,
+        debugMode);
 
-    var positionCamera = this.ellipsoid.cartographicToCartesian(coordinate);
 
-    this.layers = [];
-    this.map = null;
-
-    this.cameras = null;
     this.selectNodes = null;
     this.scheduler = Scheduler(this);
     this.orbitOn = false;
 
     this.stylesManager = new StyleManager();
 
     this.gLDebug = gLDebug;
-    this.gfxEngine = c3DEngine(this, positionCamera, viewerDiv, debugMode, gLDebug);
-    this.browserScene = new BrowseTree(this.gfxEngine);
+    this.gfxEngine = c3DEngine(this, viewerDiv, debugMode, gLDebug);
 
     this.needsRedraw = false;
     this.lastRenderTime = 0;
     this.maxFramePerSec = 60;
+    this._extents = new THREE.Box3(new THREE.Vector3(0, 0, 0), new THREE.Vector3(1, 1, 1));
```

Comment:
This doesn't seem to be used any more.

If you still want to keep it, rename to either _extends ("the scene extends...") or _extent ("the scene's extent). I favours the second option.